tests/test_index_reader.py

```diff
@@ -396,6 +398,49 @@ def test_jstring_encoding(self):
         # the discussion here: https://github.com/castorini/pyserini/issues/770
         JString('zoölogy')
 
+    def test_dump_documents_BM25(self):
+        filepath = 'collections/cacm_documents_bm25_dump.json'
+        self.index_reader.dump_documents_BM25(filepath)
+        f = open(filepath, 'r')
+        dump = json.load(f)
+        f.close()
+
+        assert(len(dump) == self.index_reader.stats()['documents'])
+
+        query = 'Where can I learn to fly a helicopter?'
```

Comment:
Where did you get these queries from?

These are topics used in the actual CACM collection:
https://github.com/castorini/anserini-data/blob/master/CACM/topics.cacm.txt

Can you change to a few queries selected from this list? Would be a better topical fit...